VirtoCommerce.Storefront/Domain/Catalog/CatalogService.cs

```diff
@@ -360,6 +360,8 @@ protected virtual Task LoadProductCustomerReviewsAsync(List<Product> products, W
                     };
                     return _customerReviewService.SearchReviews(criteria);
                 }, 1, CustomerReviewSearchCriteria.DefaultPageSize);
+
+                product.AverageRating = Convert.ToInt32(product.CustomerReviews?.Average(r => r.Rating) ?? 0);
```

Comment:
it's incorrect to calculate it in storefront, not in module

Reply:
In which module should I calculate it ? In CustomerReviews ?

Reply:
fixed